src/main/java/uk/gov/companieshouse/officerfiling/api/controller/ValidationStatusControllerImpl.java

```diff
@@ -66,7 +66,7 @@ public ValidationStatusResponse validate(
         @PathVariable("filingResourceId") final String filingResourceId,
         final HttpServletRequest request) {
 
-        logger.debugContext(transaction.getId(), "GET validation request", new LogHelper.Builder(transaction)
+        logger.debugContext(transaction.getId(), "GET validation status request", new LogHelper.Builder(transaction)
```

Comment:
Is this log still required or is it a duplicate of the interceptor log?

Reply:
There's also one at the start of uk.gov.companieshouse.officerfiling.api.controller.OfficerFilingControllerImpl#createFiling. Is this still required too?